### ISSUE TITLE
[P2][Sourcing][Accessibility][Multi input value help dialog]the button is marked with aria-haspopup=true as a menu button, which is not correct. aria-haspopup should be removed or use aria-haspopup=dialog

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10298]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
the button is marked with aria-haspopup=true as a menu button, which is not correct. aria-haspopup should be removed or use aria-haspopup=dialog

#### If this is a bug, please provide steps for reproducing it.
the button to open value help dialog on click of multi input trigger button , is marked with aria-haspopup=true as a menu button, which is not correct. aria-haspopup should be removed or use aria-haspopup=dialog

#### Is there anything else we should know?
reported by customer

**Comment by I543348** [comment]
ATL comment:
The node selector of our automation tool did not work on this dialog, hence could not find this one

**Comment by nikolay-kolarov** [comment]
Hi,

As commented in #10286 those buttons need to be hidden from the screen reader on desktop.
On mobile they will open a full screen dialog, so aria-haspopup can be removed.

Best Regards,
Nikolay

**Comment by droshev** [comment]
We need to downport to `ng15`

**Comment by mikerodonnell89** [comment]
this will be fixed by #10431

**Comment by droshev** [comment]
fixed with `0.43.7`

**Comment by sagar7259** [comment]
Hello,

This issue is still not fixed for any of the multi value inputs of desktop examples hence this ticket should be reopened. The aria-haspopup=true is still provided and due to this the value help button is read has 'Menu' which is incorrect.

@rvamudha please reopen this ticket.

Regards,
Sagar

**Comment by droshev** [comment]
If you tested Multi Input Mobile - this is not supposed to be testing on desktop. It is just to demonstrate how the component will behave on Mobile(not on desktop)

**Comment by I543348** [comment]
Hi @droshev , this is not reported for Multi input mobile mode as you can see the comment from ATL team above its tested for "desktop" examples like Simple multi input.
could you pls reopen ?
Thanks.

**Comment by droshev** [comment]
@I543348 Can you help find the multi input example where there is a popup and it is not the mobile multi input? that can help

**Comment by sagar7259** [comment]
@droshev you can find the examples for multi value input for desktop here 
https://sap.github.io/fundamental-ngx/#/core/multi-input

Below is the screenshot of the example
![image](https://github.com/SAP/fundamental-ngx/assets/143692208/2b886dc7-c635-47ce-aee3-a2c6e33dddff)

Regards,
Sagar(ATL Team)

**Comment by droshev** [comment]
@sagar7259 which version did you test that on? - it should not be on `0.45.1-rc.11`, it should be `0.43.10`

**Comment by sagar7259** [comment]
@droshev this was tested in 0.43.10 itself and the issue still persists.

Still this button is read has "Menu". Kindly look into it.

Regards,
Sagar
